culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
+
+# On Review Culture
+
+Review culture came to us from the practices of software engineers. Large
+codebases are incredibly complex: they contain the ideas and work of hundreds
+of people, manifested in millions of lines of code. They need to remain
+logically consistent, even as they grow and change. A big part of how engineers
+manage is through code review: every single change to the system is reviewed by
+at least one other person before becoming official, or "merging".
+
+Ensuring that all work is reviewed brings several benefits. Most obviously, it
+leads to higher quality work, because the final output includes feedback and
+suggestions from others. Just as importantly, reviews enable the exchange of
+knowledge, context, and perspective within the team. In a good review, both the
+contributor and the reviewer can teach each other about how work gets done on
+the project. Finally, review ensures that every part of the project is
+understood by at least two people.
+
+# The Review Process
+
+## Changes
+The review process orients around _changes_, which are the units of work being
+reviewed.
+
+A change can take many forms, for example:
+- Adding a new guide or piece of documentation
```

Comment:
Mixed feelings about all documentation going through this process. For technical documentation (e.g. SC user docs, developer docs, etc.), it makes sense. This is basically what all OSS projects do. No need to reinvent the wheel. You do need to have some control over critical documents, for security purposes sometimes (especially if money/crypto is involved), and also because some people will want "official" docs they have a higher level of confidence in. At the same time, the trend in the last few years is that much of the documentation is unofficial, crowdsourced from users in the form of StackOverflow posts, personal blog posts, forum posts, etc. Most user guides I see for larger products/projects are just individual users documenting their process to install/use something. Often these guides are collectively better than what the company can produce. We want to make sure those people get in the graph and get rewarded. Making people use GitHub might be a low barrier for OSS projects (where most use GitHub already). But once you go outside of that into other domains, that could be a significant barrier. 

For SourceCred, I think "official" documentation should live in GitHub (/docs is a good idea). But I think we should recognize that a lot of the crowdsourced documentation/guides may only surface on Discourse (or other platforms we create plugins for). Perhaps that is valued less. But it should not be zero.

Reply:
Agreeing with your points here. But to be clear, do you think the current wording suggests _all_ documentation must follow this review process?

Reply:
My reading was that it was about documentation in the context of GitHub, but you're right it could be inferred to be from elsewhere. If GitHub isn't thought of as a central place to host version controlled docs (with sources coming in from everywhere via plugins, etc.) then we probably need to clarify this.

Reply:
Updated the phrasing:

> A change can take many forms, for example:
> - Adding or changing official SourceCred documentation (in [sourcecred/docs](https://github.com/sourcecred/docs))
> - Adding or changing the SourceCred codebase